src/components/input/themes/light/input.material.scss

```diff
@@ -1,10 +1,10 @@
 @use 'sass:map';
 @use '../../../../styles/utilities' as *;
 
-$idle-color: color(gray, 600) !default;
+$idle-color: color(gray, 500) !default;
 $idle-hover-color: color(gray, 800) !default;
 $hover-background: color(gray, 200) !default;
-$focus-background: var(--focus-background, color(gray, 300)) !default;
+$focus-background: var(--focus-background, color(gray, 200)) !default;
```

Comment:
I think this change is unnecessary

-Box input:
Focus state:
-background (gray-300)

Reply:
Reverted back to the original value.